recipe/meta.yaml

```diff
@@ -9,7 +9,9 @@ source:
   sha256: 3c3e0308110c8164550e0d42ff80b8d756a70cb16c358a7cb9f57d63e87ceaf0
 
 build:
-  number: 1000
+  number: 1001
+  run_exports:
+    - blosc
```

Comment:
not sure if for semantic versioning a `- blosc` or `- {{ pin_subpackage('blosc') }}` (defaults ok) is the right way to go, [docs don't say so explicit](https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#export-runtime-requirements) how flexible the first variant might be

Reply:
We went a compatible pin here. This will be a downstream requirement for other packages and they should be able to use any compatible version. 

@conda-forge/blosc do y’all agree?

Reply:
So better `- {{ pin_subpackage('blosc') }}` then?

Reply:
I think we want pin_compatible

Reply:
I think I might have been wrong here...sorry!